mineinabyss-features/src/main/kotlin/com/mineinabyss/guilds/menus/GuildNavigation.kt

```diff
@@ -84,7 +86,7 @@ fun GuiyOwner.GuildMainMenu(player: Player) {
 }
 
 @Composable
-fun GuildUIScope.HomeScreen() {
+fun GuildUIScope.HomeScreen(feature: GuildFeature) {
```

Comment:
Instead of passing `feature` to each function, just add it to GuildUIScope and anything that uses it as a receiver (i.e. `fun GuildUIScope.something`) will get access to it more cleanly ;p